scripts/run-espresso-ci-job

```diff
@@ -20,3 +20,13 @@ gotestsum \
     -p 1 \
     ./system_tests/ \
     -run 'TestCaffNode'
+
+gotestsum \
```

Comment:
This is unnecessary because we have had this `-run 'TestEspresso'` above.

This test runs all the tests that match *TestEspresso* and this is our tradition to make every test starts with `TestEspresso`.